modules/swagger-parser-v3/src/test/resources/issue-1706/SimpleRequestResponseRef.json

```diff
@@ -0,0 +1,87 @@
+{
```

Comment:
Would need to use spaces instead of tabs

Reply:
done

Reply:
Done!

Reply:
@mma5997 it seems that json file still has tabs instead of spaces. Sorry about that, it's to get PRs as clean as we can

Reply:
Thanks for pointing it out @frantuma. 
Sorry, my bad, Eclipse formatting likes tabs it seems XD. No issues at all, I agree with you, it should be clean definitely. I have fixed it.